### ISSUE TITLE
"Error: too many mutations (max 10)" on v0.5.0

### ISSUE THREAD

**Issue by darcien** [silverbulletmd/silverbullet#523]
TL;DR silverbullet server crash on start after upgrading to v0.5.0 with error `An exception was thrown as a result of invoking function loadBuiltinsIntoIndex error: Error: too many mutations (max 10)`

## Steps

From an existing silverbullet v0.4.0

1. `silverbullet upgrade`
2. upgrade success, restart the running silverbullet server
3. see error in server logs

## Blind guess

The `Error: too many mutations (max 10)` has some hits in GH search from Deno KV.
Specifically from this KV test
https://github.com/denoland/deno/blob/9a46a824bd897e240af8a14f9d950ab6d95f42a5/cli/tests/unit/kv_test.ts#L1177

## Version and logs

```
❯ deno --version
deno 1.36.0 (release, x86_64-unknown-linux-gnu)
v8 11.6.189.12
typescript 5.1.6

❯ silverbullet version
0.5.0
```

<details><summary>server log</summary>

```log
Going to start SilverBullet binding to 127.0.0.1:3000
NOTE: SilverBullet will only be available locally (via http://localhost:3000).
To allow outside connections, pass -L 0.0.0.0 as a flag, and put a TLS terminator on top.
Running in server-processing mode, keeping state in /home/darcien/projects/pelorperak/space/.silverbullet.db
Loaded plug tasks
Loaded plug query
Loaded plug editor
Loaded plug federation
Loaded plug emoji
Loaded plug markdown
Loaded plug index
Loaded plug plug-manager
Loaded plug template
Loaded plug share
Loaded plug directive
Loaded plug sync
Loaded plug search
Indexing space for the first time (in the background)
Clearing page index...
Loading builtins attributes into index
An exception was thrown as a result of invoking function loadBuiltinsIntoIndex error: Error: too many mutations (max 10)
    at data:application/javascript;base64,dmFyIG1vZD0oKCk9Pnt2......WWUoa3IpO30pKCk7Cg==:1:1239
    at data:application/javascript;base64,dmFyIG1vZD0oKCk9Pnt2......WWUoa3IpO30pKCk7Cg==:1:1287
    at innerInvokeEventListeners (ext:deno_web/02_event.js:792:7)
    at invokeEventListeners (ext:deno_web/02_event.js:839:5)
    at dispatch (ext:deno_web/02_event.js:696:9)
    at dispatchEvent (ext:deno_web/02_event.js:1101:12)
    at pollForMessages (ext:cli/runtime/js/99_main.js:160:7)
    at eventLoopTick (ext:core/01_core.js:183:11)
error: Uncaught (in promise) Error: too many mutations (max 10)
          resultCbs && resultCbs.reject(new Error(data.error));
                                        ^
    at Sandbox.onMessage (https://get.silverbullet.md/:57597:41)
    at Worker.worker.onmessage (https://get.silverbullet.md/:57571:14)
    at Worker.wrappedHandler (ext:deno_web/02_event.js:1426:12)
    at innerInvokeEventListeners (ext:deno_web/02_event.js:792:7)
    at invokeEventListeners (ext:deno_web/02_event.js:839:5)
    at dispatch (ext:deno_web/02_event.js:696:9)
    at Worker.dispatchEvent (ext:deno_web/02_event.js:1101:12)
    at Worker.#pollMessages (ext:runtime/11_workers.js:204:12)
    at eventLoopTick (ext:core/01_core.js:183:11)
```

</details>

**Comment by zefhemel** [comment]
Please upgrade your deno version: `deno upgrade`

**Comment by darcien** [comment]
I can confirm there's no error when using Deno v1.37.1 🥳
I'll make sure to include upgrading Deno as part of the SB upgrade process.

Thanks Zef!
